.chloggen/receiver_statsdreceiver_add-uds-support.yaml

```diff
@@ -0,0 +1,21 @@
+# Use this changelog template to create an entry for release notes.
+# If your change doesn't affect end users, such as a test fix or a tooling change,
+# you should instead start your pull request title with [chore] or use the "Skip Changelog" label.
+
+# One of 'breaking', 'deprecation', 'new_component', 'enhancement', 'bug_fix'
+change_type: "bug_fix"
```

Comment:
Is there a bug that fixed by this PR? If it's just refactoring, no changelog entry is needed

Reply:
I remove it on the commit [#7094a68](https://github.com/open-telemetry/opentelemetry-collector-contrib/pull/24832/commits/7094a680995dac9e32e58c9d394fb6416fdc2a33)